packages/tfjs-node-helpers/src/testing/metrics/pr-auc.ts

```diff
@@ -0,0 +1,40 @@
+import { binarize } from '../../utils/binarize';
+import { Metric } from '../metric';
+import { MetricCalculator, MetricCalculatorData } from '../metric-calculator';
+import { PrecisionMetricCalculator } from './precision';
+import { RecallMetricCalculator } from './recall';
+
+export class PRAucMetricCalculator extends MetricCalculator {
+  constructor(private thresholdNumber: number = 21) {
+    super();
+  }
+
+  public calculate({ trueValues, probabilities }: MetricCalculatorData): Metric {
+    let area = 0;
+    let threshold = 0;
+    const step = 1 / (this.thresholdNumber - 1);
```

Comment:
@VladimirGlinskii Why do we subtract `1` from `this.thresholdNumber` here?

Reply:
Bacause we don't take the first threshold (0) for calculation of steps count. In other words stepsNumber = thresholdNumber - 1